docs/console-settings/api.md

```diff
@@ -63,17 +71,15 @@ endpoints:
 
 ::::: slot details
 
-Currently, your Console can only have a single API key (auth_token). Managing this key, as well as enabling and disabling the use of the API can be done using the following endpoints.
+Your Console supports having multiple API keys (auth_token). Managing these keys, as well as enabling and disabling the use of the API can be done using the following endpoints.
+
+API keys are created with an `Admin` or `Read-Only` role, a `Key ID` to help identify them in logs, and a `Note` to remind you of its purpose. These are fixed at creation. To change these, simply create a new API Key, rotate out the use of the old API Key, and delete the old API Key.
```

Comment:
API keys are created with an `Admin` or `Read-Only` role and a `Note` to remind you of its purpose.   These are fixed at creation. To change these, simply create a new API Key, rotate out the use of the old API Key, and delete the old API Key. Additionally a `Key ID` will be generated for each API Key that is used to help identify the key used in logs, and for use in key management endpoints.